src/smartpeak/include/SmartPeak/core/SequenceSegmentHandler.h

```diff
@@ -112,7 +112,17 @@ namespace SmartPeak
     const std::map<std::string, std::vector<OpenMS::AbsoluteQuantitationStandards::featureConcentration>>&
     getComponentsToConcentrations() const;
 
-private:
+    void setOuterComponentsToConcentrations(
```

Comment:
I recommend changing "Outer" to "Outlier" for all of the method and variable names.